build-images.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+SSM_APP_VERSION=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
+SSM_APP_VERSION=$(echo "${SSM_APP_VERSION}" | head -1)
+SSM_SERVER_IMAGE_VERSION=${SSM_APP_VERSION%-*}
+HADOOP_VERSION=3.2.4
+HADOOP_PROFILE=3.2
+CLUSTER_TYPE=$1
+
+if [ -z "$CLUSTER_TYPE" ]
+then
+  CLUSTER_TYPE=multihost
+fi
+
+echo "=============================="
+echo "      Rebuild the project     "
+echo "=============================="
+mvn clean package -Pdist,web,hadoop-${HADOOP_PROFILE} -DskipTests
```

Comment:
The new profile for building and copying compiled frontend bundle to `smart-web-server` resources is `web-ui` 
```suggestion
mvn clean package -Pdist,web-ui,hadoop-${HADOOP_PROFILE} -DskipTests
```

Reply:
fixed